src/app/components/events/overview2/overview2.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="container-fluid">
+    <div class="row align-items-start">
+        <div class="col">
+            <div class="table-responsive">
+                <h3>Overview of all events:</h3>
+                <table class="table table-hover table-striped align-middle">
+                    <thead>
+                        <tr class="table-info">
+                            <th scope="col">#</th>
+                            <th scope="col">Title</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr
+                            *ngFor="let event of events; let id = index"
+                            (click)="onSelectEvent(event)"
+                            [class.table-primary]="isEventSelected(event)"
+                        >
+                            <th scope="row">{{ id }}</th>
+                            <td>{{ event.title }}</td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+        </div>
+        <div class="col">
+            <p>Select an event in the panel on the left.</p>
+        </div>
+        <div class="col"></div>
+    </div>
+</div>
+
+<!--TODO - As suggested in the task, use the ngif Angular syntax to show the detail2 template or an alternative message. -->
```

Comment:
You should be able to figure out how to implement this with the resources described here:
https://angular.io/guide/built-in-directives#adding-or-removing-an-element-with-ngif